### PR TITLE
Integrate hedging and accounting with pazuz related screens

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Integrate accounting and hedging to pazuz admin](https://trello.com/c/gRxTCadG/995-integrate-accounting-and-hedging-to-pazuz-admin)

File: new-lamassu-admin/src/pages/Authentication/Register.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import { useQuery, useMutation } from '@apollo/react-hooks'
 import { makeStyles, Grid } from '@material-ui/core'
 import Paper from '@material-ui/core/Paper'
+// import base64 from 'base-64'
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Authentication/Register.js]: Can we remove this?

File: new-lamassu-admin/src/pages/Assets/Assets.js [hunk 2]
```diff
@@ -163,47 +133,126 @@ const formatCurrency = amount =>
...
+  const { userData } = useContext(AppContext)
+
+  const { data, loading } = useQuery(GET_OPERATOR_BY_USERNAME, {
+    context: { clientName: 'pazuz' },
+    variables: { username: userData?.username }
+  })
+
+  const operatorData = R.path(['operatorByUsername'], data)
+
+  console.log('operatorData', operatorData)
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Assets/Assets.js]: Remove this as well

**Review by josepfo — CHANGES_REQUESTED** [review]

**Comment by josepfo** [comment]
Seems that it has conflicts

**Review by josepfo — APPROVED** [review]

**Pull request merged**